Logging/composer.json

```diff
@@ -5,7 +5,7 @@
     "minimum-stability": "stable",
     "require": {
         "php": "^8.0",
-        "google/cloud-core": "^1.52.7",
+        "google/cloud-core": "^1.58",
```

Comment:
Bump the `google/cloud-core` dependency to what will be the next release, as the tests depend on the new metadata provider class.